test/rbac/suite.go

```diff
@@ -55,6 +55,7 @@ func (s *TestSuite) SetupTestSuite(c *input.Context) error {
 	}
 	s.keycloakPassword = password
 	umbrella := charts.CreateUmbrellaRelease().
-		Set("onos-config.openidc.issuer", "https://keycloak-dev.onlab.us/auth/realms/master")
+		Set("onos-config.openidc.issuer", "https://keycloak-dev.onlab.us/auth/realms/master").
+		Set("onos-config.openpolicyagent.enabled", true)
 	return umbrella.Install(true)
```

Comment:
Add `Set("onos-config.openpolicyagent.regoConfigMap", "onos-umbrella-opa-rbac")` to point at my new REGO rules and when they are in place the tests pass